### PR TITLE
Don't rerun gerald for title/body edits to the PR

### PR DESCRIPTION
**Pull request by jaredly**
## Summary:
<Add your summary here>

Issue: XXX-XXXX

## Test plan:

### CODE REVIEW

File: .github/workflows/gerald-pr.yml [hunk 1]
```diff
@@ -7,8 +7,8 @@ name: Gerald - Notify and Request Reviewers On Pull Request
     types: [opened, synchronize, reopened, ready_for_review, edited]
 jobs:
   gerald:
-    # Skip this workflow on dependabot PRs
-    if: github.actor != 'dependabot[bot]' && github.actor != 'dependabot-preview[bot]'
+    # Skip this workflow on dependabot PRs. Don't re-run if only the title or body changes
+    if: github.actor != 'dependabot[bot]' && github.actor != 'dependabot-preview[bot]' && (github.event.action != 'edited' || github.event.changes.base != null)
```
Comment (jeresig) [inline .github/workflows/gerald-pr.yml:11]: Ahh - can we not put this into the shared action?

**Review by jeresig — APPROVED** [review]
Thanks for the fixes!

**Review by somewhatabstract — APPROVED** [review]
This is great. Thanks, @jaredly.

I also wonder the same thing as @jeresig; is this something we can include in the shared action(s)?

**Comment by jaredly** [comment]
So, the shared action wouldn't allow us to "not run the workflow at all". It's possible we could do something like that with reusable workflows, but I haven't played with those yet https://docs.github.com/en/actions/using-workflows/reusing-workflows

**Pull request closed without merging**